Core/include/Scene.h

```diff
@@ -8,6 +8,14 @@ class Scene
     Scene();
     ~Scene();
 
+    Entity* getRoot() {
+       return sceneRoot;
+    };
+    
+    void setRoot(Entity* newRoot) {
+       sceneRoot = newRoot;

```

Comment:
Not sure if we ever want to set a new root, it should be an empty entity all the time